### DISCUSSION TITLE
Bootstrapping a Blitz app with the latest Canary version available on github

### DISCUSSION THREAD

**Discussion by cloudcompute** [blitz-js/blitz#1887] [General] [answered]
Hello folks

This project is awesome. I have several questions. For now, it is about begin working with blitz. I want to do this:

a. Create a new application with the latest Blitz version currently available. The latest is 0.30.6 and when I run 'blitz new my-app' it says that you can work with 0.29.7. How can I start using the Blitz 0.30.6, kindly tell me.

b. Start working on the app.

c. Upgrade my working application directory with the latest blitz changes that the wonderful Blitz team pushes on github. Can this simply be done "git pull". 

Thanks
Raman

**Comment by flybayer** [comment, marked as answer]
@ramandhingra you can use the canary version like this:

1. `npm i -g blitz@canary`
2. `blitz new myApp`

That will generate a new app with the latest code from canary!

**Reply by cloudcompute** [reply]
@flybayer Thanks for the reply
